touchnet/utils/inference.py

```
@@ -0,0 +1,124 @@
+            "help": ("workers for dataloader"),
+        },
+    )
+    prefetch: int = field(
+        default=8,
+        metadata={
+            "help": ("prefetch for dataloader"),
+        },
+    )
+
+
+class AudioDataset(Dataset):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)